### ISSUE TITLE
requesting that a license be created for wxSwift

### ISSUE THREAD

**Issue by KimTheLearner** [wxSwift/wxSwift#9]
To the owner(s) of this repository (@wxSwift):

I would like to request that a LICENSE.txt file be created in the root directory of this project, that would grant permissions to copy and modify this code.  

Why?  I would really like to create a fork of this code and work on it. 

But evidently I should not, without a license.  I just read the following:

**The most common misconception about open-source projects is if a Github project has no license then it’s okay to use. If a repository has no license, then all rights are reserved and it is not open-source or free. You cannot modify or redistribute this code without explicit permission from the copyright holder.**  -- https://levelup.gitconnected.com/code-on-github-is-not-free-197f93e4cd44

One way, I understand, is for there to be a LICENSE.txt file in the root directory that states the license.

I did see the LICENSE file in the wxc subdirectory, but that would not cover the rest of the source code in the repo.

So, could one of the repository owners please create a suitable LICENSE file in the root directory?

@tinysun212, I see you have contributed much in the past.  Are you able to assist with this?

Thanks!

**Comment by melizasw** [comment]
I agree with @gandalfas that this project should be released under a permissive/libre license if possible. At present it is usable as nothing but curiosity and prevents others from continuing this work or from making use of the name "wxSwift" in a different project. Unless this project is opened up or deleted it is effectively abandon-ware that will only create confusion with any other effort to bring Swift language bindings to wxWidgets.

@tinysun212 - You are clearly the creator and sole owner of this project. Please chime in and know what your intentions for this project are. Thanks.
